consensus/src/ordered_broadcast/mod.rs

```diff
@@ -658,13 +696,193 @@ mod tests {
+                let context = context.with_label(&validator.to_string());
+                let monitor = mocks::Monitor::new(111);
+                monitors.insert(validator.clone(), monitor.clone());
+                let sequencers = mocks::Sequencers::<PublicKey>::new(vec![sequencer.public_key()]);
+                let validators = mocks::Validators::<PublicKey>::new(
+                    identity.clone(),
+                    validator_pks.clone(),
+                    Some(share.clone()),
+                );
+
+                let automaton = mocks::Automaton::<PublicKey>::new(|_| false);
+                automatons
+                    .lock()
+                    .unwrap()
+                    .insert(validator.clone(), automaton.clone());
+
+                let (reporter, reporter_mailbox) = mocks::Reporter::<Ed25519, Sha256Digest>::new(
+                    namespace,
+                    *poly::public(&identity),
+                    Some(5),
+                );
+                context.with_label("reporter").spawn(|_| reporter.run());
+                reporters.insert(validator.clone(), reporter_mailbox);
+
+                let engine = Engine::new(
+                    context.with_label("engine"),
+                    Config {
+                        crypto: scheme.clone(),
+                        relay: automaton.clone(),
+                        automaton: automaton.clone(),
+                        reporter: reporters.get(validator).unwrap().clone(),
+                        monitor,
+                        sequencers,
+                        validators,
+                        namespace: namespace.to_vec(),
+                        epoch_bounds: (1, 1),
+                        height_bound: 2,
+                        rebroadcast_timeout: Duration::from_secs(5),
+                        priority_acks: false,
+                        priority_proposals: false,
+                        journal_heights_per_section: 10,
+                        journal_replay_concurrency: 1,
+                        journal_name_prefix: format!("ordered-broadcast-seq/{}/", validator),
+                        journal_compression: Some(3),
+                    },
+                );
+
+                let ((a1, a2), (b1, b2)) = registrations.remove(validator).unwrap();
+                engine.start((a1, a2), (b1, b2));
+            }
+
+            // Spawn sequencer engine
+            {
+                let context = context.with_label("sequencer");
+                let automaton = mocks::Automaton::<PublicKey>::new(|_| false);
+                automatons
+                    .lock()
+                    .unwrap()
+                    .insert(sequencer.public_key(), automaton.clone());
+                let (reporter, reporter_mailbox) = mocks::Reporter::<Ed25519, Sha256Digest>::new(
+                    namespace,
+                    *poly::public(&identity),
+                    None,
```

Comment:
If we want other sequencers to process/verify messages from other sequencers, we should set this to the same constraint.